package.json

```diff
@@ -7,8 +7,8 @@
   },
   "devDependencies": {
     "eslint": "^7.15.0",
-    "purescript-psa": "^0.8.0",
-    "pulp": "^15.0.0",
+    "purescript-psa": "^0.8.2",
+    "pulp": "16.0.0-0",
```

Comment:
What's the reason for the prerelease version of Pulp? Should this be `16.0.0`?

Reply:
I made a pre-release of `16.0.0` in case we wanted to add support for esbuild later or come across other bugs.